### DISCUSSION TITLE
How to query JSON objects  in Isar collection?

### DISCUSSION THREAD

**Discussion by bahman2000** [isar/isar#1485] [Help] [answered]
Suppose we have a collection with the following columns: 

String data1;
String data2;

Example data:

data1: `"test1"`
data2:  `"{
"name":"John",
"age":30,
"cars":"["Ford", "BMW", "Fiat"]",
"info": "{"salary":56000,"married": true }"
}"`

 The query that is needed:

`collection.data2.info.married == true`

**Comment by basselalsayed** [comment, marked as answer]
If data2 has a consistent structure it's probably easiest to mark data2 as an `@embedded` class instead of json then you'll get all query functionality.
